content/blog/monitor-application-performance-across-hybrid-cloud-environment-using-apache-skywalking-and-service-mesh.md

```diff
@@ -0,0 +1,320 @@
+
+We can check the detailed Apache SkyWalking installation by typing the following *kubectl* command:
+
+```markdown
+$ kubectl get all -n skywalking
+NAME                                                 READY   STATUS      RESTARTS   AGE
+pod/elasticsearch-master-0                           1/1     Running     0          6h34m
+pod/elasticsearch-master-1                           1/1     Running     0          6h34m
+pod/skywalking-oap-init-n92hp                        1/1     Completed   0          88s
+pod/skywalking-skywalking-helm-oap-bfb57fbf8-27frm   1/1     Running     0          92s
+pod/skywalking-skywalking-helm-oap-bfb57fbf8-djzw5   1/1     Running     0          52s
+pod/skywalking-skywalking-helm-ui-7776f4854d-nvkds   1/1     Running     0          6h34m
+
+NAME                                     TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)               AGE
+service/elasticsearch-master             ClusterIP   172.20.171.140   <none>        9200/TCP,9300/TCP     6h34m
+service/elasticsearch-master-headless    ClusterIP   None             <none>        9200/TCP,9300/TCP     6h34m
+service/skywalking-skywalking-helm-oap   ClusterIP   172.20.155.177   <none>        11800/TCP,12800/TCP   6h34m
+service/skywalking-skywalking-helm-ui    ClusterIP   172.20.205.170   <none>        80/TCP                6h34m
+
+NAME                                             READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/skywalking-skywalking-helm-oap   2/2     2            2           6h34m
+deployment.apps/skywalking-skywalking-helm-ui    1/1     1            1           6h34m
+
+NAME                                                        DESIRED   CURRENT   READY   AGE
+replicaset.apps/skywalking-skywalking-helm-oap-bfb57fbf8    2         2         2       96s
+replicaset.apps/skywalking-skywalking-helm-ui-7776f4854d    1         1         1       6h34m
+
+NAME                                    READY   AGE
+statefulset.apps/elasticsearch-master   2/2     6h34m
+
+NAME                            COMPLETIONS   DURATION   AGE
+job.batch/skywalking-oap-init   1/1           31s        94s
+```
+
+We change the service types for both service *skywalking-skywalking-helm-oap*   and *skywalking-skywalking-helm-ui*  from *ClusterIP* to *LoadBalancer*. With the built-in support of Elastic Load Balancing in AWS EKS cluster, the external accessible load balancing host names will be created for those two services in Apache SkyWalking installation.
```

Comment:
You can change the service type for both.....
With the built-in support of Elastic Load Balancing (ELB) in AWS EKS cluster, ....